build-setup/pytorch/setup_pytorch.md

```diff
@@ -0,0 +1,42 @@
+# Setup for Pytorch
+
+If you wish to build and run the PyTorch inference applications you will need to install Pytorch and build the torch libraries. 
+
+Although not strictly required it is recommended to install [Anaconda](https://www.anaconda.com/products/individual#download-section) as the following instructions use the Conda Package manager that comes with Anaconda. 
+
+
+## Install the Python Dependencies 
+```
+conda install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses
+```
+
+## Clone the v1.7.1 release of the Pytorch repository
+```
+git clone --depth=1 --branch=v1.7.1 git@github.com:pytorch/pytorch.git
+cd pytorch
+# update submodules used by PyTorch
+git submodule sync
+git submodule update --init --recursive
+``` 
+
+
+## Build macOS
+```
+export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
```

Comment:
There's an assumption here that CMake is available, but it's not on a newly installed Mac, so instructions for installing CMake should be in these instructions.  Probably before the `conda install` step in case installing the Python `cmake` module requires that the `cmake` binary be on the path.